int8 RECORD_TYPE_ALL=0
int8 RECORD_TYPE_SNAPSHOT=1
int8 RECORD_TYPE_RECORD=2
int8 RECORD_TYPE_THUMB=3
int8 RECORD_TYPE_SCHED_RECORD=4

#record file descriptor
string id
string name
uint32 dur
int8 type
time create
uint32 size